src/Models/HydrostaticFreeSurfaceModels/SplitExplicitFreeSurfaces/split_explicit_timesteppers.jl

```diff
@@ -0,0 +1,151 @@
+"""
+    struct ForwardBackwardScheme
+
+A timestepping scheme used for substepping in the split-explicit free surface solver.
+    
+The equations are evolved as follows:
+```julia
+ηᵐ⁺¹ = ηᵐ - Δτ (∂x Uᵐ + ∂y Vᵐ)
```

Comment:
is there a `g H` missing here?

Reply:
The`g H` is in the barotropic velocity equations.